src/components/Input/Input.tsx

```diff
@@ -55,6 +62,15 @@ export const Input = ({ sticky }: { sticky: boolean }) => {
     }
   }, [inputRef, input]);
 
+  useEffect(() => {
+    if (attachments.length > 0) {
+      trackEvent('hl_chat_attachments_present', { 
+        attachmentCount: attachments.length,
+        attachmentTypes: attachments.map(a => a.type)
+      });
+    }
+  }, [attachments]);
```

Comment:
This would fire every time you add attachments no? Which we are already sending events for? Perhaps better to send this info when you actually submit a message?

Reply:
Good catch! yes :D

Reply:
Done